### PR TITLE
Added remaining references from my master's thesis

### PR DESCRIPTION
**Pull request by h3ssto**
Remaining references from my master's thesis.

### CODE REVIEW

File: literature.bib [hunk 1]
```diff
@@ -12022,6 +12183,25 @@ @inproceedings{KTS+:ICSE09
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2008 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{HHT:CPAIOR08,
+	title = {{Propagating separable equalities in an MDD store}},
```
Comment (pmbittner) [inline literature.bib:12187]: capitalization

File: literature.bib [hunk 2]
```diff
@@ -15489,6 +15728,18 @@ @incollection{R03
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2002 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{M:TC02,
+	title = {{Streaming BDD manipulation}},
```
Comment (pmbittner) [inline literature.bib:15732]: M?

File: literature.bib [hunk 3]
```diff
@@ -16322,6 +16595,27 @@ @book{S01
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2000 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{DG:VLSI00,
+	author = "Drechsler, Rolf and G{\"u}nther, Wolfgang",
```
Comment (pmbittner) [inline literature.bib:16599]: use { }

File: literature.bib [hunk 4]
```diff
@@ -16322,6 +16595,27 @@ @book{S01
...
+	title = {{History-Based Dynamic Minimization During BDD Construction}},
+	bookTitle=VLSI,
+	publisher = Kluwer,
+	address = Deventer,
+	pages= {334--345},
+	doi = "10.1007/978-0-387-35498-9_30",
```
Comment (pmbittner) [inline literature.bib:16605]: {}

File: literature.bib [hunk 5]
```diff
@@ -17051,7 +17437,61 @@ @inproceedings{KLMMLLI97
...
+	author={Sanghavi, Jagesh V and Ranjan, Rajeev K and Brayton, Robert K and Sangiovanni-Vincentelli, Alberto},
+	booktitle=DAC,
+	pages={635--640},
+	publisher = ACM,
+	address = NY,
+	year=1996
+}
+
+@inproceedings{NJF+:ICCAD96,
+	title={Partitioned ROBDDs - A Compact, Canonical and Efficiently Manipulable Representation for Boolean Functions},
```
Comment (pmbittner) [inline literature.bib:17453]: double brackets

File: literature.bib [hunk 6]
```diff
@@ -17354,12 +17816,36 @@ @article{WF94
...
+	address = Amsterdam,
+	year = 1993
+}
+
+@inproceedings{OYY:ICCAD93,
+	author = {H. Ochi and K. Yasuoka and S. Yajima},
+	title = {{Breadth-First Manipulation of Very Large Binary Decision Diagrams}},
+	booktitle = ICCAD,
+	pages = {48--55},
+    organization=IEEE,
```
Comment (pmbittner) [inline literature.bib:17838]: publisher?

File: literature.bib [hunk 7]
```diff
@@ -17570,6 +18056,49 @@ @article{M92
...
+	booktitle = EURODAC,
+	pages = {50--54},
+	publisher = IEEE,
+	address = Washington,
+	year = 1991,
+}
+
+@inproceedings{BRKM:DAC91,
+	title={{Heuristics to Compute Variable Orderings for Efficient Manipulation of Ordered Binary Decision Diagrams}},
+	author={Butler, Kenneth M and Ross, Don E and Kapur, Rohit and Mercer, M Ray},
```
Comment (pmbittner) [inline literature.bib:18082]: Put dots after abbreviated names?
`Butler, Kenneth M. and Ross, Don E. and Kapur, Rohit and Mercer, M. Ray`

File: literature.bib [hunk 8]
```diff
@@ -17622,6 +18151,20 @@ @incollection{HV91
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 1990 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{FS:TC90,
+	author = {S.J. Friedman and K.J. Supowit},
```
Comment (pmbittner) [inline literature.bib:18155]: Could you find out the first names?

File: literature.bib [hunk 9]
```diff
@@ -18026,6 +18588,39 @@ @article{S81
...
 
+@article{APT:IPL79,
+	title = {{A Linear-Time algorithm for Testing the Truth of Certain Quantified Boolean Formulas}},
+	author = {Bengt Aspvall and Michael F. Plass and Robert Endre Tarjan},
+	journal = IPL,
+	volume = "8",
+	number = "3",
+	pages = {121--123},
+	doi = "https://doi.org/10.1016/0020-0190(79)90002-4",
+	keywords = "quantified boolean formula, strongly connected components, 2-CNF, 2-satisfiability",
```
Comment (pmbittner) [inline literature.bib:18599]: brackets for all entries here

File: literature.bib [hunk 10]
```diff
@@ -18026,6 +18588,39 @@ @article{S81
...
+	author = {Garey, Michael R and Johnson, David S},
+	volume = {174},
+	publisher= Springer,
+	address = Cham,
+	year = 1979
+}
+
+@article{V:TCS79,
+	title = {{The Complexity of Computing the Permanent}},
+	author = {Valiant, Leslie G},
```
Comment (pmbittner) [inline literature.bib:18614]: G.

File: literature.bib [hunk 11]
```diff
@@ -18730,6 +19376,14 @@ @misc{JADE
 	note = {Accessed: 2020-04-21}
 }
 
+@misc{JEDD,
+	title = {JEDD: Source Code},
+	howpublished = {\url{http://www.sable.mcgill.ca/jedd/}},
+	note = {Accessed: 2021-03-01},
+	year =
```
Comment (pmbittner) [inline literature.bib:19383]: incomplete entry. Looks like a syntax error.

**Review by pmbittner — CHANGES_REQUESTED** [review]
All minor comments except for one syntax error.

File: literature.bib (hunk 7 above)
Comment (h3ssto) [reply literature.bib:18082]: Not according to DBLP or the BitTeX docs.

File: literature.bib (hunk 8 above)
Comment (h3ssto) [reply literature.bib:18155]: Good catch.

File: literature.bib (hunk 9 above)
Comment (h3ssto) [reply literature.bib:18599]: Does not change anything but ok.

File: literature.bib (hunk 10 above)
Comment (h3ssto) [reply literature.bib:18614]: Nopes.

File: literature.bib (hunk 11 above)
Comment (h3ssto) [reply literature.bib:19383]: Fixed.

File: MYabrv.bib [hunk 12]
```diff
@@ -383,12 +413,15 @@ @String{SEI
...
-	@String{Secaucus = "Secaucus, NJ, USA"}
+    @String{BerlinHeidelberg = "Berlin, Heidelberg"}
+    % Sprinter International Publishing AG (old: Birkhäuser)
+    @String{Basel = "Basel, Switzerland"}
+    @String{Cham = "Cham, Switzerland"}
+    @String{Denver = "Denver, CO, USA"} % also for ACM
+    @String{Dordrecht = "Dordrecht, The Netherlands"}
+    @String{London = "London, UK"}
+    @String{Secaucus = "Secaucus, NJ, USA"}
+    @String{Wiesbaden = "Wiesbaden"}
```
[the remark this answers is not in the document]
Comment (h3ssto) [reply MYabrv.bib:424]: Not mine, but ok.

File: literature.bib (hunk 3 above)
Comment (h3ssto) [reply literature.bib:16599]: Same, but oki.

**Comment by h3ssto** [comment]
All resolved in 2947782

**Pull request merged**